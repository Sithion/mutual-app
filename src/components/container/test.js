import { View } from 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Container from './container';

test('render "Container" correctly', () => {
  const container = renderer.create(
    <Container>
      <View />
    </Container>).toJSON();
  expect(container).toMatchSnapshot();
});
