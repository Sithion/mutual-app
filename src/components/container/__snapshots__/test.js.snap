// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render "Container" correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#CACACA",
      "bottom": 0,
      "flexDirection": "column",
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View />
</View>
`;
